envoy-control-core/src/main/resources/lua/ingress_client_name_header.lua

```diff
@@ -0,0 +1,25 @@
+function envoy_on_request(handle)
+    local streamInfo = handle:streamInfo()
+    local trusted_header = handle:metadata():get("trusted_client_identity_header") or ""
```

Comment:
`ingressClientNameHeaderFilter()` will return actual filter only if we set `trustedClientIdentityHeader` - so this check is just here in case of misconfiguration?

Reply:
No, it's neccessary, because even if `trustedClientIdentityHeader` is set to non-empty string, the metadata  is not set for example for `/status/*` routes.